### ISSUE TITLE
Inconsistent file sorting order

### ISSUE THREAD

**Issue by VADemon** [vatterspun/cubicexplorer#4]
It's either really inconsistent or just not the same as what Windows Explorer is doing.

**Example 1:**
Go to a folder with many files and directories (e.g. Windows TEMP folder). Turn on "Detail List" view mode. Click on column "Date Modified" to sort.

CE: If sorted "↓" then most recent **files** appear on top. If ↑ then the oldest **folders** appear on top.

Windows Explorer: ↓ = newest folders are on top. ↑ Oldest folders are on top.

- Basically in Windows Explorer the folders always come BEFORE displaying files. CE deviates from that.

**Example 2:**

Files with numbers in the name are not sorted properly. Especially this breaks keyboard typing out the name to jump to the files. The numbers in file name may be negative, 0, positive, zero prefixed positive numbers (00023.png). This may lead a person to think that the file doesn't exist, unless they try CTRL+F search.

**Comment by vatterspun** [comment]
I've noticed a similar issue.  Great suggesiton
